File: Auto-generated-tasks/Difficulty-1/Task-3/Solution.f90
! Version of fortran: 90

integer:: Num = 6, u, A2 = 0, p
integer:: Mas(Num)
call random_seed;
do u=0,Num
    call random_number(r)
    p = (419 * r) + 4
    if(p>422) p = 422
    Mas(u) = p
end do
print *, "A2=", A2
end

File: Auto-generated-tasks/Difficulty-1/Task-3/Solution.pas
Var
	Num:=6, u, A2:= 0: integer;
	Mas:Array[0..Num] of integer;
Begin
	for u:= 0 to Num do begin
		Mas[u] := Random(418) + 4;
		A2 := A2 + Mas[u];
	end;
	WriteLn("A2=", A2);
End.